### PR TITLE
refactor forms styles

### PR DESCRIPTION
**Pull request by moran711**

### CODE REVIEW

File: src/components/model-form/model-form.styles.js [hunk 1]
```diff
@@ -1,68 +1,39 @@
...
+    TEXT_FIELD_STYLES,
+    CONTROLS_BLOCK_STYLES,
+    TABS_STYLES,
+    LARGE_STYLES,
+    ATTACH_FILE_STYLES,
+    IMAGE_UPLOAD_STYLES,
+    ITEM_UPDATE_STYLE,
+    INPUT_ERROR_STYLES,
+    IMAGE_UPLOAD_CONTAINER_STYLES,
+    IMAGE_NAME_STYLE
```
Comment (vasylshpak) [inline src/components/model-form/model-form.styles.js:18]: Nice to see that everything is put together now! The only thing I recognize is that maybe we should have "STYLES" like everywhere else.

```suggestion
    IMAGE_NAME_STYLES
```

File: src/components/news-form/news-form.styles.js [hunk 2]
```diff
@@ -1,33 +1,25 @@
...
-    backgroundColor: 'white',
-    '& span.MuiTab-wrapper': {
-      color: '#3F51B5'
+export const useStyles = makeStyles((theme) => {
+  const {
+    DETAILS_STYLES,
+    SAVE_BUTTON_STYLES,
+    TABS_STYLES,
+    CONTROLS_BLOCK_STYLES,
+    ITEM_UPDATE_STYLE,
```
Comment (vasylshpak) [inline src/components/news-form/news-form.styles.js:10]: the same here: 
```suggestion
    ITEM_UPDATE_STYLES
```

**Review by vasylshpak — CHANGES_REQUESTED** [review]

File: src/components/model-form/model-form.styles.js (hunk 1 above)
Comment (moran711) [reply src/components/model-form/model-form.styles.js]: Fixed

File: src/components/news-form/news-form.styles.js (hunk 2 above)
Comment (moran711) [reply src/components/news-form/news-form.styles.js]: Fixed

**Review by vasylshpak — APPROVED** [review]

File: src/components/pattern-form/pattern-form.styles.js [hunk 3]
```diff
@@ -1,68 +1,39 @@
...
-    backgroundColor: 'white',
-    '& span.MuiTab-wrapper': {
-      color: '#3F51B5'
+export const useStyles = makeStyles((theme) => {
+  const {
+    DETAILS_STYLES,
+    INPUT_ERROR_STYLES,
+    IMAGE_NAME_STYLES,
+    SAVE_BUTTON_STYLES,
+    IMAGE_UPLOAD_CONTAINER_STYLES,
```
Comment (nadiyafomenko) [inline src/components/pattern-form/pattern-form.styles.js:10]: All objects with styles are lowercased, why do we need this uppercase?

Comment (moran711) [reply src/components/pattern-form/pattern-form.styles.js:10]: Fixed

**Review by nadiyafomenko — APPROVED** [review]

**Review by Iryna-Bzdyr — APPROVED** [review]

**Pull request merged**